src/main/java/com/iridium/iridiumskyblock/commands/BoostersCommand.java

```diff
@@ -54,6 +54,16 @@ public boolean execute(CommandSender sender, String[] args) {
                             islandBooster.setTime(LocalDateTime.now().plusSeconds(booster.time));
                             IslandLog islandLog = new IslandLog(island.get(), LogAction.BOOSTER_PURCHASE, user, null, 0, boosterName);
                             IridiumSkyblock.getInstance().getDatabaseManager().getIslandLogTableManager().addEntry(islandLog);
+
+                            player.sendMessage(
+                                StringUtils.color(
+                                    IridiumSkyblock.getInstance().getMessages().successfullyBoughtBooster
+                                        .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)
+                                        .replace("%booster%", IridiumSkyblock.getInstance().getBoosterList().get(islandBooster.getBooster()).item.displayName)
```

Comment:
Just set this to entry.key with some Capitalisation magic?

Reply:
We talked about this on Discord, I guess? Are you fine with this solution?